packages/core/bootstrap/src/lib/external-adapter/README.md

```diff
@@ -67,6 +67,55 @@ const base = validator.validated.data.base
 const quote = validator.validated.data.quote
 ```
 
+### overrideToken
+
+#### Arguments
+
+- `symbol` (String): The symbol to get the token of
+- `network` (String): Optional param to define which network to get the token for. Default: "mainnet"
+
+Use `overrideToken` to get the token address of the given symbol
+
+```typescript
+const address = validator.overrideToken("WETH")
+// "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2"
+```
+
+### overrideIncludes
+
+#### Arguments
+
+- `adapter` (String): The name of the adapter in use
+- `from` (String): The "from" symbol
+- `to` (String): The "to" symbol
+- `includes` (Includes[]): Array of Includes objects
+
+#### Includes Type:
+
+```typescript
+type Includes = {
+    from: string, // From symbol
+    to: string, // To symbol
+    adapters?: string[], // Array of adapters this applies to
+    inverse?: boolean // If the inverse should be calculated instead
+    tokens?: boolean // If the token addresses should be used instead
+}
+```
```

Comment:
I think we should change the order so we make the explanation for overrideIncludes a bit clearer

edit: removed suggested code change due to github formatting

Reply:
Github completely messed up the formatting here :sweat_smile: 
My point was that we should first document the Includes type, then we document overrideIncludes